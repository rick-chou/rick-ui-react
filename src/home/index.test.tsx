import React from 'react';
import { shallow } from 'enzyme';
import Home from './index';

describe('Home components', () => {
  it('basic', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
