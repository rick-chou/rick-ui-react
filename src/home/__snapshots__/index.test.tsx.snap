// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home components basic 1`] = `
<div
  className="px-4 py-16 mx-auto sm:max-w-xl md:max-w-full lg:max-w-screen-xl md:px-24 lg:px-8 lg:py-20"
>
  <div
    className="grid gap-10 lg:grid-cols-2"
  >
    <div
      className="lg:pr-10"
    >
      <h5
        className="mb-4 text-4xl font-extrabold leading-none"
      >
        Home ðŸ”¥
      </h5>
      <p
        className="mb-6 text-gray-900"
      />
      <hr
        className="mb-5 border-gray-300"
      />
      <div
        className="flex items-center space-x-4"
      >
        <a
          className="text-gray-600 transition-colors duration-300 hover:text-deep-purple-accent-400"
          href="https://github.com/rick-chou"
          target="_blank"
        >
          <IconGithub />
        </a>
        <a
          className="text-gray-600 transition-colors duration-300 hover:text-deep-purple-accent-400"
          href="https://juejin.cn/user/1574156383563496"
          target="_blank"
        >
          <IconJueJin />
        </a>
        <a
          className="text-gray-600 transition-colors duration-300 hover:text-deep-purple-accent-400"
          href="https://luckychou.gitbook.io/blog"
          target="_blank"
        >
          <IconBlog />
        </a>
      </div>
    </div>
    <div>
      <img
        alt=""
        className="object-cover w-full h-56 rounded shadow-lg sm:h-96"
        src="https://cdn.jsdelivr.net/gh/rick-chou/rick-assets/jpg/38.jpg"
      />
    </div>
  </div>
</div>
`;
